parlai/core/dict.py

```diff
@@ -333,12 +333,13 @@ def add_additional_special_tokens(self, additional_special_tokens: List[str]):
         """
         self.additional_special_tokens = additional_special_tokens
 
-        if (
-            self.additional_special_tokens
-            and not self.supports_additional_special_tokens()
-        ):
-            raise RuntimeError(
-                f'{self.tokenizer} does not currently support adding additional special tokens'
+        if hasattr(self, 'bpe'):
```

Comment:
not true. you can add extra special tokens with split or space

Reply:
see lines you deleted below

Reply:
They weren't properly handled.

Reply:
With space or split, special tokens were still being split up

Reply:
Okay, supported now.